modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -74,24 +74,6 @@ public int getWeight() {
 		return _weight;
 	}
 
-	public boolean hasParameterTypes(
```

Comment:
@nicolas-sss, please fix the commit message: `Remove unused code`.

In addition, is this method no longer used in any place?

Reply:
Yes, it was the last use of it, the previous commit. So became unnecessary.